src/api/public/apidocs-new/components/responses/search/published_binary_id.yaml

```diff
@@ -0,0 +1,126 @@
+description: OK. The request has succeeded.
+content:
+  application/xml; charset=utf-8:
+    schema:
+      type: object
+      properties:
+        matches:
+          type: integer
+          xml:
+            attribute: true
+        limited:
+          type: boolean
+          xml:
+            attribute: true
+        binary:
+          type: array
+          items:
+            type: object
+            properties:
+              name:
+                type: string
+                xml:
+                  attribute: true
+              project:
+                type: string
+                xml:
+                  attribute: true
+              package:
```

Comment:
@eduardoj sorry, for the confusion, there is a `package` in the schema but an example is missing :)

Reply:
Right, they were missing in the examples. :+1: Added.